producer-model-mapper/src/main/scala/za/co/absa/spline/producer/modelmapper/v1/spark/SparkSplineOperationOutputConverter.scala

```diff
@@ -31,6 +31,7 @@ class SparkSplineOperationOutputConverter(
     attributeDefs
       .groupBy(_ (FieldNamesV1.AttributeDef.Id).toString)
       .mapValues(_.head)
+      .view.force // see: https://github.com/scala/bug/issues/4776
```

Comment:
This is just an idea.

Maybe we could define common `mapValueForce` function and use it everywhere as a workaround. So the problem would be documented on one place.